test/componentarrays.jl

```diff
@@ -93,6 +93,40 @@ soln(0.9)
 #     ob.color = findnode(soln(t), :C)[point_map]
 # end
 
+function closest_point(p1, p2, dims)
+  p_res = collect(p2)
+  for i in 1:length(dims)
+    if dims[i] != Inf
+      p = p1[i] - p2[i]
+      f, n = modf(p / dims[i])
+      p_res[i] += dims[i] * n
+      if abs(f) > 0.5
+        p_res[i] += sign(f) * dims[i]
+      end
+    end
+  end
+  Point3{Float64}(p_res...)
+end
+
+function flat_op(s::AbstractDeltaDualComplex2D, X::AbstractVector; dims=[Inf, Inf, Inf])
```

Comment:
Delete

Reply:
Can't this is used in line: https://github.com/AlgebraicJulia/Decapodes.jl/blob/aea31c661808c5d1aed571af70c1bc88bb3d0bf7/test/componentarrays.jl#L152

Reply:
Removing these should become an issue and a separate PR. I would consider removing this not in the scope of this PR.